src/hearings/containers/request-hearing/hearing-create-edit-summary/hearing-create-edit-summary.component.ts

```diff
@@ -1,28 +1,62 @@
-import { Component, OnDestroy } from '@angular/core';
+import { Component, OnDestroy, OnInit } from '@angular/core';
 import { Store } from '@ngrx/store';
 import { ACTION, Mode } from '../../../models/hearings.enum';
 import { HearingsService } from '../../../services/hearings.service';
 import * as fromHearingStore from '../../../store';
 import { HEARING_CREATE_EDIT_SUMMARY_TEMPLATE } from '../../../templates/hearing-create-edit-summary.template';
 import { RequestHearingPageFlow } from '../request-hearing.page.flow';
+import { string } from '@pact-foundation/pact/src/dsl/matchers';
+import { Section } from 'src/hearings/models/section';
+import { ScreenNavigationModel } from 'api/hearings/models/screenNavigation.model';
```

Comment:
Should not have references to src, should be '../'. Also, preferably should not be using api/ but not present in src then not as necessary